modules/hal_nordic/nrf_802154/CMakeLists.txt

```diff
@@ -94,6 +94,8 @@ set(NRF53_SERIES  ${CONFIG_SOC_SERIES_NRF53X})
 set(SER_HOST      ${CONFIG_NRF_802154_SER_HOST})
 set(SL_OPENSOURCE ${CONFIG_NRF_802154_SL_OPENSOURCE})
 
+set(NRF_802154_CARRIER_FUNCTIONS_ENABLED ${CONFIG_NRF_802154_CARRIER_FUNCTIONS})
```

Comment:
Shouldn't it be done using `target_compile_definitions` syntax like other configuration macros above?